packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -80,7 +76,7 @@ export class HelmChart extends Construct {
 
     const stack = Stack.of(this);
 
-    const provider = HelmResourceProvider.getOrCreate(this);
+    const provider = KubectlProvider.getOrCreate(this);
```

Comment:
Will all users understand that `KubectlProvider` can also handle Helm charts? Is that common/logical in k8s world?

Reply:
Hmm I guess most won't need to know huh.

Reply:
It's an internal class